### PR TITLE
responsive layout, first page

### PR DESCRIPTION
**Pull request by VictorRadoi**
responsive layout added and some parts from first page

### CODE REVIEW

File: website/lib/presentation/desktop/home_page/home_page_desktop.dart
Comment (VictorRadoi) [inline website/lib/presentation/desktop/home_page/home_page_desktop.dart:1]: fnf

File: new_flutter_website/.fvmrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (LucaDorinAnton) [inline new_flutter_website/.fvmrc:1]: Can we call this folder `flutter`?

File: new_flutter_website/.gitignore
Comment (LucaDorinAnton) [inline new_flutter_website/.gitignore:1]: include to root `.gitignore`, 
also please add to it:
- .vscode
- android
- ios
- linux
- macos
- windows 
- website
- web

also please remove from a PR (I can't review a PR with 300 files in it)

(I tried running the build without those and it worked)

**Review by LucaDorinAnton — CHANGES_REQUESTED** [review]
Please clean up PR so that I can review it thoroughly

**Review by LucaDorinAnton — APPROVED** [review]

**Pull request merged**